lib/quarantine.rb

```diff
@@ -4,6 +4,23 @@
 require 'quarantine/databases/base'
 require 'quarantine/databases/dynamo_db'
 
+module RSpec
+  module Core
+    class Example
+      # The implementation of clear_exception in rspec-retry doesn't work
+      # for examples that use `it_behaves_like`, so we implement our own version that
```

Comment:
Do we know if this is an issue that the `rspec-retry` maintainers are aware of and will fix?

Reply:
I'm not sure. But I think it's OK to fix it in `quarantine` because

1. it's maintained by us so we don't have to depend on the author of `rspec-retry` to merge the PR
1. What we want here is a way to skip a failed test, we don't have to depend on `rspec-retry` to do that.